onyo/lib/commands.py

```diff
@@ -471,57 +450,47 @@ def onyo_get(inventory: Inventory,
              match: list[Callable[[dict], bool]] | None = None,
              keys: list[str] | None = None,
              sort: dict[str, sort_t] | None = None) -> list[dict]:
-    r"""Query the repository for information about assets.
+    r"""Query the key-values of assets.
 
     Parameters
     ----------
     inventory
-      The inventory to query.
+        The Inventory to query.
     include
-      Limits the query to assets underneath these paths.
-      Paths can be assets and directories.
-      If no paths are specified, the inventory root is used as default.
+        Assets or directories to query.
```

Comment:
`onyo get --path shelf warehouse/desktop_dell_D11S.G7MPSM2 --keys path` works just fine, i.e. one can mix assets and directories.

So I think instead of "Assets *or* directories" this should say "Assets *and* directories"

Reply:
This is a nitpick, because `or` is ambiguous in english. You're reading it as `xor`.

I have changed this (and a dozen other instances of this, across the project) to `and/or` for the sake of explicitness.